### PR TITLE
Refactor handlers

### PR DESCRIPTION
**Pull request by samedii**
Mostly refactor of handlers to ignite-style.

Rather than attaching default handlers we are now using a constructor function `create_standard_trainer_validator` that puts all the pieces together in the correct (and not always obvious) order. Ignite uses a similar style with the public function `create_supervised_trainer`

### CODE REVIEW

File: workflow/ignite/handlers/early_stopping.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import ignite
+from ignite.engine import Events
+
+from workflow.ignite.tqdm_print import tqdm_print
+
+
+class EarlyStopping:
+    def __init__(self, score_function, trainer, config):
+        self.early_stopping_handler = ignite.handlers.EarlyStopping(
```
Comment (Jim-Holmstroem) [inline workflow/ignite/handlers/early_stopping.py:9]: indentation wonkyness

File: workflow/ignite/create_standard_trainer_validator.py [hunk 2]
```diff
@@ -79,3 +82,5 @@ def add_default_event_handlers(
     )
 
     EarlyStopping(model_score_function, trainer, config).attach(validator)
+
+    return trainer, validator
```
Comment (Jim-Holmstroem) [inline workflow/ignite/create_standard_trainer_validator.py]: newline endoffile

**Review by Jim-Holmstroem — APPROVED** [review]

File: workflow/ignite/handlers/early_stopping.py (hunk 1 above)
Comment (samedii) [reply workflow/ignite/handlers/early_stopping.py:9]: Good catch :)

File: workflow/ignite/create_standard_trainer_validator.py (hunk 2 above)
Comment (samedii) [reply workflow/ignite/create_standard_trainer_validator.py]: Need something that does this automatically....

Comment (Jim-Holmstroem) [reply workflow/ignite/create_standard_trainer_validator.py]: an IDE? ;)

**Pull request merged**